src/Sulu/Bundle/AdminBundle/Resources/js/components/Select/Option.js

```diff
@@ -29,7 +29,7 @@ export default class Option extends React.PureComponent<Props> {
         focus: false,
         selected: false,
         selectedVisualization: 'icon',
-        value: '',
+        value: undefined,
```

Comment:
This would be the default, and doesn't have to be separately defined.

Reply:
fixed 🙂